### PR TITLE
docs: update code of conduct

### PR DESCRIPTION
**Pull request by abhisar-yadav**
Corrected typos in the Code of Conduct File

### How has the user experience changed?
These changes improve the readability

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@abhisar-yadav 

The original wording is mostly taken from the original template on https://www.contributor-covenant.org/version/1/4/code-of-conduct/ and this is generally considered to be correct.

Whether or not your suggestions should be incorporated is more of a style judgement than anything else.

If your PR is going to be accepted you would need to sign the CLA agreement and  correct the semantic commit message. `docs:` would be the right prefix here.

```text
Error: No release type found in pull request title "Update CODE_OF_CONDUCT.md". Add a prefix to indicate what kind of release this pull request corresponds to. Cypress types are:/

Available types:
 - breaking: A breaking change that will require a MVB
 - dependency: A change to a dependency that impact the user
 - deprecation: A API deprecation notice for users
 - feat: A new feature
 - fix: A bug or regression fix
 - misc: Misc user-facing changes, like a UI update, which is not a fix or enhancement to how Cypress works
 - perf: Changes that improves performance
 - chore: Changes to the build process or auxiliary tools and libraries such as documentation generation
 - docs: Documentation only changes
 - refactor: A code change that neither fixes a bug nor adds a feature that is not user-facing
 - revert: Reverts a previous commit
 - test: Adding missing or correcting existing tests
```

**Comment by abhisar-yadav** [comment]
@MikeMcC399 I have signed the CLA agreement and changed the semantic commit message to docs

**Comment by MikeMcC399** [comment]
@abhisar-yadav 

> I have signed the CLA agreement and changed the semantic commit message to docs

You probably need to have a space after `docs:` and before `Update`.

**Comment by abhisar-yadav** [comment]
@MikeMcC399  any other changes you would want me to make??

**Comment by MikeMcC399** [comment]
@abhisar-yadav 

> any other changes you would want me to make??

You need to pay attention to the workflow check https://github.com/cypress-io/cypress/actions/runs/6429590135/job/17458964061?pr=27988 which requires your commit, not just the PR title, to also conform to semantic messaging.

This is not what I'm asking for. It's what the PR process asks for. 🙂 

Edit: Sorry for a wrong link. I corrected this.

**Review by jennifer-shehane — APPROVED** [review]
Thanks for the edits!

**Review by dkasper-was-taken — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity, and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (MikeMcC399) [inline CODE_OF_CONDUCT.md]: Please check if the change of 
"gender identity and expression"
to
"gender identity, and expression",
by the addition of a comma, is correct.

"gender identity and expression" are not separable items of a list, rather it is one fixed term, similar to the later term in the same sentence "sexual identity and orientation" where no additional comma has been proposed.

The master template on https://www.contributor-covenant.org/version/1/4/code-of-conduct/ has no comma in this place, nor in the current version https://www.contributor-covenant.org/version/2/1/code_of_conduct/ so I would tend to think that the proposed addition of a comma here is incorrect.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One of the proposed changes of adding a comma is incorrect.

**Comment by MikeMcC399** [comment]
@abhisar-yadav 

- If this PR were to be implemented it would change only one of the `CODE_OF_CONDUCT.md` files in the [cypress-io](https://github.com/cypress-io/) organization making them inconsistent. That does not seem to be desirable.

- I have submitted https://github.com/cypress-io/cypress/issues/28003 to suggest creating a central `CODE_OF_CONDUCT.md` file.

**Pull request merged**